### ISSUE TITLE
Error on opening Vitashell C2-12828-1 Saved core file succeeded. ux0:data/psp2core-*********-eboot.bin.psp2dump

### ISSUE THREAD

**Issue by gthind92** [TheOfficialFloW/VitaShell#534]
I am currently running 3.65 firmware on PS Vita 1000. i was able to install henkaku and vitashell using hencore but when i launched Vitashell, got the error "C2-12828-1 Saved core file succeeded. ux0:data/psp2core-*********-eboot.bin.psp2dump"
I tried all basic troubleshooting(Restarts, rebuild database, format memory card and system) and searched all over web but no luck. i cannot access vita files because molecular and vitashell does not work. I do not have any backups as i just bought it.

![vita error](https://user-images.githubusercontent.com/47749886/52960379-e6dd2480-3355-11e9-9d2b-2a38053204cf.jpg)

Please help

**Comment by rey-k2** [comment]
hey i'm having the same problem here. have you found a solution?

**Comment by MegaPramuk** [comment]
You _are_ in HEN before trying to open VitaShell, right?
Either by h-encore or Ensō.
Assuming you are when launching, C2-12828-1 is a file related error (corruption/not found). Meaning, redownloading VitaShell via h-encore _should_ solve it.

**Comment by rey-k2** [comment]
> You _are_ in HEN before trying to open VitaShell, right?
> Either by h-encore or Ensō.
> Assuming you are when launching, C2-12828-1 is a file related error (corruption/not found). Meaning, redownloading VitaShell via h-encore _should_ solve it.

yeah henkaku settings are shown and on system info it has 3.65 and the characters next to it. i restored vitashell through psvimgtools using qcma, but i've tried redownloading vitashell through h-encore before and i still get that error

**Comment by Phantasmal-Groove** [comment]
I have the same problem on 3.73 firmware, can't open VitaShell at all, after installing HENkaku same error C2-12828-1 appears.

**Comment by rey-k2** [comment]
yeah i ended up having to buy another motherboard and replacing it :/ 
sorry dude
